src/Util/FilterUtil.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('FilterUtil', () => {
   describe('featureMatchesFilter', () => {
     it('returns true if a feature matches the filter', () => {
       const dummyData = TestUtil.getComplexGsDummyData();
-      let matchingFeature = dummyData.exampleFeatures.features[0];
+      const matchingFeature = dummyData.exampleFeatures.features[0];
```

Comment:
Just out of curiosity: Shouldn't `const` be ok here, as we keep the reference to the same object?

Reply:
`const` is not only ok but recommended by eslint. Thats why i switched :)